Std/Data/Int/Basic.lean

```diff
@@ -109,6 +107,13 @@ This pair satisfies `div x y = round_to_zero (x / y)`.
 `Int.div` and `Int.mod` are defined in core lean.
 -/
 
+/--
+Core Lean provides instances using T-rounding division, i.e. `Int.div` and `Int.mod`.
+We override these here.
+-/
+instance (priority := 1001) : Div Int := ⟨Int.ediv⟩
+instance (priority := 1001) : Mod Int := ⟨Int.emod⟩
```

Comment:
```suggestion
instance : Div Int := ⟨Int.ediv⟩
instance : Mod Int := ⟨Int.emod⟩
```
We don't need higher priority here since you cannot import this file without importing the core file first, so this one will always win.